rocksdb_admin/admin_handler.cpp

```diff
@@ -2182,4 +2222,41 @@ std::vector<std::string> AdminHandler::getAllDBNames() {
     return db_manager_->getAllDBNames();
 }
 
+void AdminHandler::setS3Config(
+    const std::string& s3_bucket,
+    const std::string& s3_backup_dir_prefix,
+    const std::string& snapshot_host_port,
+    const uint32_t limit_mbs,
+    const bool include_meta) {
+  backup_manager_->setS3Config(std::move(s3_bucket), std::move(s3_backup_dir_prefix), 
+    std::move(snapshot_host_port), limit_mbs, include_meta);
+}
+
+bool AdminHandler::checkS3Object(
```

Comment:
What is the goal from this function, can we move it into backup manager as well?